documentation/modules/ROOT/pages/crafting-cluster-telco-related-infra-operators-configs.adoc

```diff
@@ -93,11 +92,91 @@ spec:
+      metadata:
+        annotations:
+          kubeletconfig.experimental: |
+            {"topologyManagerScope": "pod",
+             "systemReserved": {"memory": "3Gi"}
+            }
+      spec:
+        cpu:
+          isolated: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-cpu-isolated" (index .ManagedClusterLabels "hardware-type")) hub}}'
+          reserved: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-cpu-reserved" (index .ManagedClusterLabels "hardware-type")) hub}}'
+        hugepages:
+          defaultHugepagesSize: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-hugepages-default" (index .ManagedClusterLabels "hardware-type"))| hub}}'
+          pages:
+          - count: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-hugepages-count" (index .ManagedClusterLabels "hardware-type")) | toInt hub}}'
+            size: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-hugepages-size" (index .ManagedClusterLabels "hardware-type")) hub}}'
+        numa:
+          topologyPolicy: single-numa-node
+        realTimeKernel:
+          enabled: false
+        globallyDisableIrqLoadBalancing: false
+        # WorkloadHints defines the set of upper level flags for different type of workloads.
+        # The configuration below is set for a low latency, performance mode.
+        workloadHints:
+          realTime: true
+          highPowerConsumption: false
+          perPodPowerManagement: false
+    - fileName: TunedPerformancePatch.yaml
+      policyName: "group-policy"
+      spec:
+        profile:
+          - name: performance-patch
+            data: |
+              [main]
+              summary=Configuration changes profile inherited from performance created tuned
+              include=openshift-node-performance-openshift-node-performance-profile
+              [sysctl]
+              # When using the standard (non-realtime) kernel, remove the kernel.timer_migration override from the [sysctl] section
+              # kernel.timer_migration=0
+              [scheduler]
+              group.ice-ptp=0:f:10:*:ice-ptp.*
+              group.ice-gnss=0:f:10:*:ice-gnss.*
+              [service]
+              service.stalld=start,enable
+              service.chronyd=stop,disable
 EOF
 -----
 +
-2. Since we are deploying DUs, we need to run the validator policies crafted by the CNF team as well:
+2. We're using policy templating, so we need to create the `ConfigMap` with the templating values to be used. Notice that the following resource contains values for multiple hardware specifications that we may have in our infrastructure. In the policy we have just applied we are obtaining the value for tuning our SNO by obtaining the value of the label `hardware-type` each cluster was assigned to. This label was set in the `SiteConfig` definition in the link:crafting-deployments-iaac.html#siteconfig[previous section].
```

Comment:
It might be useful to include a link to ACM's hub templating documentation, if possible (under Governance > Template processing).